tests/tracking/fluent/test_fluent.py

```diff
@@ -357,6 +357,91 @@ def test_list_experiments(view_type, tmpdir):
         process.terminate()
 
 
+def test_search_experiments(tmp_path):
+    sqlite_uri = "sqlite:///{}".format(tmp_path.joinpath("test.db"))
+    store = SqlAlchemyStore(sqlite_uri, default_artifact_root=str(tmp_path))
+
+    num_all_experiments = SEARCH_MAX_RESULTS_DEFAULT + 1  # +1 for the default experiment
+    num_active_experiments = SEARCH_MAX_RESULTS_DEFAULT // 2
+    num_deleted_experiments = SEARCH_MAX_RESULTS_DEFAULT - num_active_experiments
+
+    with store.ManagedSessionMaker() as session:
+        # Create active experiments
+        active_experiment_names = [f"active_{i}" for i in range(num_active_experiments)]
+        active_experiments = [
+            SqlExperiment(
+                name=name,
+                lifecycle_stage="active",
+                artifact_location=str(tmp_path.joinpath(name)),
+            )
+            for name in active_experiment_names
+        ]
+        # Create deleted experiments
+        deleted_experiment_names = [f"deleted_{i}" for i in range(num_deleted_experiments)]
+        deleted_experiments = [
+            SqlExperiment(
+                name=name,
+                lifecycle_stage="deleted",
+                artifact_location=str(tmp_path.joinpath(name)),
+            )
+            for name in deleted_experiment_names
+        ]
+        session.add_all(active_experiments + deleted_experiments)
+        # Create some tags
+        experiment_tags = [
+            SqlExperimentTag(key="tag", value="x", experiment_id=1),
+            SqlExperimentTag(key="tag", value="x", experiment_id=2),
+            SqlExperimentTag(key="tag", value="y", experiment_id=3),
+        ]
+        session.add_all(experiment_tags)
```

Comment:
Rather than using `store` directly to populate experiment content, can we set up the test by calling existing MLflow fluent / MLflow client APIs?

Reply:
Compared the performance between the current approach and using `mlflow.create_experiment`:

- The existing approach: took 2.23s
- Using `mlflow.create_experiment`: took 5.89s

The difference is small. I'll update the code to use `mlflow.create_experiment` :)

Reply:
Updated :)